sdk/appconfiguration/app-configuration/README.md

```diff
@@ -1,6 +1,10 @@
 # Azure App Configuration client library for JS
 
-This package contains an isomorphic SDK for ConfigurationClient.
+Azure App Configuration is a managed service that helps developers centralize their application configurations simply and securely.
+
+Modern programs, especially programs running in a cloud, generally have many components that are distributed in nature. Spreading configuration settings across these components can lead to hard-to-troubleshoot errors during an application deployment. Use App Configuration to securely store all the settings for your application in one place.
+
+Use the client library for App Configuration to create and manage application configuration settings.
 
 ## Getting started
 
```

Comment:
I can't click on the line, but actually we support Node.js version 8.x.x or higher, not 6